provider/adapters/src/fork/mod.rs

```diff
@@ -6,7 +6,7 @@
 //!
 //! # Types of Forking Providers
 //!
-//! ## marker-Based
+//! ## Mcararker-Based
```

Comment:
Mcararker? Caramaraker? Ramcarkerer?

Reply:
lol how?! I've noticed that VSCode sometimes corrupts files on large rename tasks, but where does the c from 🤔